frontend/src/metabase-lib/Question.ts

```diff
@@ -353,6 +353,18 @@ class QuestionInner {
       // TODO: move to StructuredQuery?
       const aggregations = query.aggregations();
       const breakouts = query.breakouts();
+      const sourceQuery = query.sourceQuery();
+
+      if (
+        aggregations.length === 0 &&
+        breakouts.length === 0 &&
+        sourceQuery &&
+        !this.isDataset()
```

Comment:
I'm not sure why this needs to check that the query isn't a model?

Reply:
> models always have to have display: "table"

Source: https://metaboat.slack.com/archives/C505ZNNH4/p1686833371108729?thread_ts=1686832713.430189&cid=C505ZNNH4

Reply:
I made the condition explicit in this commit: 4cc5669708807bfd748e98f144db2d086d8c206d

```tsx
      if (this.isDataset()) {
        return this.setDisplay("table");
      }
```